<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Platforms</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QListWidget" name="platformsLW"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Manage</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>168</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="nameLE">
           <property name="placeholderText">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="platformCombo">
           <property name="placeholderText">
            <string>Select Platform</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="passwordLE">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="generateTB">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="showPasswordCB">
             <property name="text">
              <string>Show Password</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="createPB">
               <property name="text">
                <string>Create</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="updatePB">
               <property name="text">
                <string>Update</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deletePB">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionChange_Master_Password"/>
    <addaction name="actionChange_Name"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuManage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChange_Master_Password">
   <property name="text">
    <string>Change Master Password</string>
   </property>
  </action>
  <action name="actionChange_Name">
   <property name="text">
    <string>Change Name</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
